common/src/main/java/com/microsoft/identity/common/internal/cache/BrokerOAuth2TokenCache.java

```diff
@@ -180,6 +181,7 @@ public BrokerOAuth2TokenCache(@NonNull Context context,
     public ICacheRecord save(@NonNull AccountRecord accountRecord,
                              @NonNull IdTokenRecord idTokenRecord,
                              @NonNull AccessTokenRecord accessTokenRecord,
+                             @NonNull RefreshTokenRecord refreshTokenRecord,
```

Comment:
I would prefer to see new API + tests added for this, rather than have to increment the _major_ version of common to change this API